Gruntfile.js

```diff
@@ -35,9 +35,10 @@ module.exports = function(grunt) {
           options: {
             sourceMaps: false,
             presets: ['@babel/preset-env'],
-            "plugins": [
-              "transform-object-rest-spread",
-              "transform-remove-strict-mode"
+            plugins: [
+              // "transform-remove-console",
+              // "transform-object-rest-spread",
```

Comment:
Are the tests working when this is removed? There was a file that used spread operators and some browsers throw an error if it doesn't have support. I had the issue, I wonder if it works now?